package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "develop": "gatsby develop",
     "start": "yarn develop",
-    "build": "NODE_OPTIONS='--max-old-space-size=4096' gatsby build",
+    "build": "NODE_OPTIONS='--max-old-space-size=8192' gatsby build",
```

Comment:
we should apply this change over all the repositories? base.instore, cea.instore, etc?

Reply:
@gtkatakura We might